### ISSUE TITLE
Error with proportion column in KeyATM()

### ISSUE THREAD

**Issue by soubhikbarari** [keyATM/keyATM#28]
I ran a `keyATM` on just a subset of 5000 docs and I got this error:

```
> model.keyATM <- keyATM(    
  docs              = keyATM_docs,
  no_keyword_topics = NUM_TOPICS,
  keywords          = KEYWORDS,
  model             = "cov", 
  model_settings    = list(covariates_data = data.matrix(stm_dfm$meta),
                           covariates_formula = ~ as.factor(meetingType))
)
Initializing the model...
Fitting the model. 1500 iterations...
0%   10   20   30   40   50   60   70   80   90   100%
[----|----|----|----|----|----|----|----|----|----|
**************************************************|
Creating an output object. It may take time...
Error: Column `Proportion` must be length 40 (the number of rows) or one, not 47
```
This was an error internal to `keyATM()` - any idea what this might be about?

**Comment by Shusei-E** [comment]
Thanks for reporting the bug! It seems this happens when some of the topics do not appear at all in the fitted object.

```r
> fitted$Z <- lapply(fitted$Z, function(x){ x[x==1] <- 2 ; return(x)})
> keyATM_output(model)  # this is an internal function
Creating an output object. It may take time...
Error: Column `Proportion` must be length 6 (the number of rows) or one, not 7
```

It seems 7 topics are not used at all in your case, which is extremely rare. Does the likelihood look fine (it should not be `nan`)?

**Comment by Shusei-E** [comment]
I fixed the code to deal with the error in this branch: https://github.com/keyATM/keyATM/tree/patch-GI28

This path simply adds blank rows if any topics do not appear, so it does not dig into why some topics are not used at all.

@tomoya-sasaki Any thoughts? Pull request: https://github.com/keyATM/keyATM/pull/30

**Comment by Shusei-E** [comment]
Pull request is merged: #30
